.buildkite/release-package-registry-distribution.yml

```diff
@@ -1,6 +1,37 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GOLANG_VERSION: '1.20.4'
+  DOCKER_REGISTRY: 'docker.elastic.co'
+
 steps:
-  - label: ":golangci-lint: Checks"
+  - input: "Input version for docker tag"
+    fields:
+      - text: "The docker tag to be published (format: major.minor.patch(-prerelease)?)."
+        hint: "e.g. 0.0.1"
+        key: "DOCKER_TAG"
+        required: true
+        default: "latest"
```

Comment:
by default it is latest , but if someone run it in UI and  erase that field (it will already in the field) - pipeline do not start. and offer to input some value